### PR TITLE
feat: add change note color functionality

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
- Note Change Color Functionality added.
- Close Button Added.
## Deploy Preview Link: https://deploy-preview-6--devdeadviz-my-notes.netlify.app/

### CODE REVIEW

**Review by dusariudaykumar — APPROVED** [review]
- The code looks good to me
- The UI looks good
- The class naming convention is on point. Good work!

File: src/reducers/noteReducer.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 const noteReducer = (state, action) => {
   switch (action.type) {
     case "ADD_NOTE":
-      return { ...state, newNotes: action.payload };
+      return { ...state, newNotes: action.payload, noteColor: "" };
```
Comment (Chiragmodi01) [inline src/reducers/noteReducer.js:4]: I'd suggest give empty string to color initially, so no need to pass it here.

**Review by Chiragmodi01 — COMMENTED** [review]
- In the UI, you 'create note form' takes data of prev edited note. Even when i create a new note.
- If i edit a note and change color but cancel it, the color stays there along with note's data. So, a fresh note will look good to user while creating new.

Overall UI looks good.

File: src/reducers/noteReducer.js (hunk 1 above)
Comment (devdeadviz) [reply src/reducers/noteReducer.js:4]: I am giving it empty so clear the value of `noteColor` when the note is added successfully.

**Comment by devdeadviz** [comment]
> * In the UI, you 'create note form' takes data of prev edited note. Even when i create a new note.
> * If i edit a note and change color but cancel it, the color stays there along with note's data. So, a fresh note will look good to user while creating new.
> 
> Overall UI looks good.

Okay, will look into it.
Thanks for pointing it out.

**Pull request merged**